src/definitions/utils.ts

```diff
@@ -0,0 +1,8 @@
+// https://stackoverflow.com/a/53395649/4573065
+type AllOf<T> = ['Needs to be all of', T];
```

Comment:
Please also export this, so if someone really needs it, they can use it
```suggestion
export type AllOf<T> = ['Needs to be all of', T];
```